get-new-tool-versions/parsers/verify-added-to-image/verify-python-parser.psm1

```diff
@@ -0,0 +1,21 @@
+function Search-PythonVersionsNotOnImage {
+    param (
+        [string]$ToolName,
+        [string]$ReleasesUrl,
+        [string]$FilterParameter,
+        [string]$FilterArch
+    )
+    
+    $stableReleases = (Invoke-RestMethod $ReleasesUrl) |
+        Where-Object stable -eq $true |
+        ForEach-Object {$_.$FilterParameter.split(".")[0,1] -join"."} |
+        Select-Object -Unique
+    $toolsetUrl = "https://raw.githubusercontent.com/actions/virtual-environments/main/images/win/toolsets/toolset-2019.json"
```

Comment:
let's check windows-2022 as it's the latest one
```suggestion
    $toolsetUrl = "https://raw.githubusercontent.com/actions/virtual-environments/main/images/win/toolsets/toolset-2022.json"
```

Reply:
Done